receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_windows.go

```diff
@@ -68,23 +68,25 @@ func startSampling(_ context.Context, logger *zap.Logger) error {
 	var err error
 	samplerInstance, err = newSampler(logger)
 	if err != nil {
-		return err
+		// To keep the same behavior, as previous versions on Windows, error in this case is just logged
+		// and the scraper is not started.
+		logger.Error("Failed to init performance counters, load metrics will not be scraped", zap.Error(err))
```

Comment:
Shouldn't we set skipScrape to true here?

Reply:
Well observed, it was on the code removed on `load_scraper.go` - I will add it soon.

Reply:
[3745ca7](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39916/commits/3745ca7513f57c8af374a2738e8f287c554c94c8)